src/os/bluestore/BlueFS.cc

```diff
@@ -378,7 +378,6 @@ int BlueFS::_write_super()
   ::encode(crc, bl);
   assert(bl.length() <= get_super_length());
   bl.append_zero(get_super_length() - bl.length());
-  bl.rebuild();
```

Comment:
Yes, rebuild is not identical to rebuild_aligned. But in there, for aio_write it don't depend on the input parameter bufferlist whether is contiguous. Other place which call aio_write don't call rebuild.
In fact, rebuild don't consider bufferlist whether already contiguous. Why dose call rebuild? It want data in a contiguous memroy.